bids-validator/src/schema/expressionLanguage.ts

```diff
@@ -43,4 +43,7 @@ export const expressionFunctions = {
     return list.filter((x) => x === val).length
   },
   exists: exists,
+  substr: (arg: string, start: int, end: int): str => {
+    return arg.substr(start, end)
```

Comment:
> `string.substr(start, length)`

https://www.w3schools.com/jsref/jsref_substr.asp

```suggestion
    return arg.substr(start, end - start)
```